database/init.sql

```diff
@@ -1,102 +1,66 @@
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (22, 'Costerd', 'Ricofort', 'Ricofort', '096-776-9198', 'Male', 'fcosterdl@baidu.com', '96564 Cooper Corner Apt. 352 Port Floy, OR 86049', 69978, '2003-07-26');
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (23, 'Corrin', 'East Graycefurt', 'East Graycefurt', '(583)403-4746', 'Male', 'tcorrinm@intel.com', '353 Israel Streets Jedediahport, GA 99481', 49614, '1988-02-29');
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (24, 'Bunford', 'West Joeport', 'West Joeport', '(245)726-8274x48974', 'Female', 'nbunfordn@joomla.org', '9755 Kshlerin Brooks East Roger, ND 23843-8553', 16353, '1979-11-25');
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (25, 'Lumley', 'Oceanestad', 'Oceanestad', '06965723793', 'Female', 'hlumleyo@w3.org', '9877 Kaia Village New Angelomouth, KS 82353-9742', 70534, '1985-06-15');
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (26, 'Whiles', 'Creolashire', 'Creolashire', '800.294.1751x13357', 'Male', 'pwhilesp@amazon.co.jp', '229 Derrick Village Gayview, OR 63688-9938', 58634, '1990-09-30');
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (27, 'Presdee', 'North Ernestinaton', 'North Ernestinaton', '556-111-2276x003', 'Female', 'mpresdeeq@sourceforge.net', '4353 Bayer Lights East Bentonville, GA 61468-6552', 90902, '1985-11-01');
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (28, 'Bedberry', 'Jakaylaland', 'Jakaylaland', '1-078-468-7156', 'Female', 'jbedberryr@taobao.com', '51526 Stamm Garden Apt. 560 Hahnview, CA 29074-8976', 10160, '1998-09-04');
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (29, 'Danilchev', 'North Esta', 'North Esta', '1-244-938-3948', 'Female', 'sdanilchevs@addtoany.com', '34989 Kuphal Inlet Suite 190 Gutkowskiville, IA 69417', 40221, '1992-10-04');
-INSERT INTO `users` (`id`, `first_name`, `last_name`, `city`, `phone`, `gender`, `email`, `address`, `postcode`, `date_of_birth`) VALUES (30, 'Whaplington', 'West Breanabury', 'West Breanabury', '1-045-399-1032x67023', 'Female', 'cwhaplingtont@cloudflare.com', '488 Martine Villages Bernadettetown, AR 39587-0766', 23031, '1998-04-05');
+# creating user instances, todelete
```

Comment:
Make sure you get rid of these in the future, or place them in another file like "mockData.sql" that is run after ALL tables + sprocs have been built

Reply:
Having a separate mockData.sql file for testing sounds like a good idea, I'll do that next issue!